src/sst/core/serialization/serializable_base.h

```diff
@@ -208,9 +208,9 @@ public:
     {                                                                                     \
         return SST::Core::Serialization::serializable_builder_impl<obj>::static_cls_id(); \
     }                                                                                     \
-    static obj* construct_deserialize_stub()                                              \
+    static obj* construct_deserialize_stub() /* NOLINT(bugprone-macro-parentheses) */     \
     {                                                                                     \
-        return new obj;                                                                   \
+        return new obj; /* NOLINT(bugprone-macro-parentheses) */                          \
```

Comment:
I got `clang-tidy` warnings about macro arguments requiring parentheses, but with inline comments, I suppressed most, if not all, of them. They cannot be used here because they change the meaning of the code.